Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java

```diff
@@ -217,6 +218,22 @@ public void shouldFindByMetadataContent() throws DataException {
         assertEquals(processNotFound, 1, processService.findByAnything("SecondMetaShort").size());
     }
 
+    @Test
+    public void shouldFindProcessWithUnderscore() throws DataException, DAOException {
+        Project projectDTO = ServiceManager.getProjectService().getById(1);
```

Comment:
Why is this variable called `projectDTO`? It is an instance of `Project`, not `ProjectDTO`.